test/bin/test-citgm.js

```diff
@@ -2,13 +2,17 @@
 
 var test = require('tap').test;
 
-var spawn = require('../../lib/spawn');
+var spawn = require('child_process').spawn;
```

Comment:
the purpose of `lib/spawn` is to wrap spawn to be cross platform compatible. If we are missing something would it not make sense to fix it in `lib/spawn`?